run_unittests.py

```diff
@@ -100,15 +67,34 @@ def setup_backend():
     os.environ['MESON_UNIT_TEST_BACKEND'] = be
     sys.argv = filtered
 
+def import_test_cases(suite: unittest.TestSuite) -> list:
+    '''
+    Imports all test classes into the current module and returns their names
+    '''
+    classes = set()
+    for test in suite:
+        if isinstance(test, unittest.TestSuite):
+            classes.update(import_test_cases(test))
+        elif isinstance(test, unittest.TestCase):
+            import importlib
```

Comment:
Is there a reason to not import this at the top level?

Reply:
no, thanks